app/integrations/discord/interactions/resolvers/ping.rb

```diff
@@ -0,0 +1,28 @@
+module Discord
+  module Interactions
+    module Resolvers
+      class Ping < Resolver
+        EXPECTED_GLOBAL_NAME = 'Discord'.freeze
```

Comment:
let's rename this to `DISCORD_GLOBAL_NAME`, and probably this value should not belong to this class, it looks like it have a bigger scope. maybe the `Resolver` class is a better place?

```suggestion
        DISCORD_GLOBAL_NAME = 'Discord'.freeze
```

Reply:
Oh I just saw we are using this to check the `global_name` of the user who does this ping call. Maybe the `User` class is a better place for this constant. WDYT?

Reply:
I agree, no it's there